apps/src/authoredHints.js

```diff
@@ -183,19 +183,11 @@ AuthoredHints.prototype.updateLightbulbDisplay_ = function (animate) {
     this.promptIcon.parentNode.insertBefore(this.lightbulb, this.promptIcon);
   }
 
-  // If there are more than nine hints, simply display "9+"
-  var hintText = (hintCount > 9) ? "9+" : hintCount;
-  if (hintCount === 0) {
-    this.lightbulb.innerHTML = lightbulbDimSVG;
-  } else {
-    this.lightbulb.innerHTML = lightbulbSVG;
-    this.lightbulb.querySelector('#hintCount').textContent = hintText;
-  }
-
-  var bulb = document.getElementById("bulb");
-  if (animate && bulb) {
-    bulb.setAttribute('class', 'animate-hint');
-  }
+  React.render(React.createElement(Lightbulb, {
+    count: hintCount,
+    lit: hintCount > 0,
+    animate: animate,
+  }), this.lightbulb);
```

Comment:
Note: Our build system actually supports JSX inside of .js files now. Unfortunately our linter does not yet. I'm going to try to do that this sprint, which would allow the preferred:
```
React.render(<Lightbulb count={hintCount} lit={hintCount > 0} animate={animate}/>, this.lightbulb).
```
Your current approach is of course fine/necessary until I fix the linter.